app/views/customers/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> Name: <%= @customer.name %> </h1>
+<h3> Add Items: </h3> 
+<%= form_with url: "/customers/#{@customer.id}", method: :get, local: true do |form| %>
+      <div>
+        <%= form.label :search, 'Add a new item by item ID number' %>
+        <%= form.text_field :search %>
+      </div>
+      <div>
+        <%= form.submit 'Submit' %>
+      </div>
+    <% end %>
+<h3> Items: </h3> 
+<ul> 
+  <% @customer.items.each do |item| %>
+    <p><%= item.name %></p>
+    <p><%= item.price %></p>
+    <p><%= item.supermarket %></p>
+  <% end %>
```

Comment:
Yeah, nice job here iterating through the items and displaying their information.